samples/snippets/src/main/java/com/example/bigquery/QueryWithStructsParameters.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_query_params_structs]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.QueryParameterValue;
+import com.google.cloud.bigquery.TableResult;
+import java.util.HashMap;
+import java.util.Map;
+
+public class QueryWithStructsParameters {
+
+  public static void queryWithStructsParameters() {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      QueryParameterValue booleanValue = QueryParameterValue.bool(true);
+      QueryParameterValue stringValue = QueryParameterValue.string("test-stringField");
+      QueryParameterValue integerValue = QueryParameterValue.int64(10);
+      // Create struct
+      Map<String, QueryParameterValue> struct = new HashMap<>();
+      struct.put("booleanField", booleanValue);
```

Comment:
nit: Can you inline these?
```suggestion
      struct.put("booleanField", QueryParameterValue.bool(true));
```